Source/Scene/Model.js

```diff
@@ -1077,8 +1079,35 @@ define([
                             var position = primitives[m].attributes.POSITION;
                             if (defined(position)) {
                                 var accessor = gltfAccessors[position];
-                                var aMin = Cartesian3.fromArray(accessor.min, 0, aMinScratch);
-                                var aMax = Cartesian3.fromArray(accessor.max, 0, aMaxScratch);
+                                var extensions = accessor.extensions;
+                                var accessorMin = new Matrix(accessor.min);
+                                var accessorMax = new Matrix(accessor.max);
+                                
+                                // If this accessor is quantized, we should use the decoded min and max
+                                if (defined(extensions)) {
+                                    var quantizedAttributes = extensions.WEB3D_quantized_attributes;
+                                    if (defined(quantizedAttributes)) {
```

Comment:
> What is the rationale in the extension spec for storing min/max quantized? Do you think it is the right approach? What is the benefit? Size is not important here. Won't almost all clients just decode it right away?

I think the rationale is that the accessor min and max should reflect the data that is actually in the buffer, which I do tend to agree with.

However, I think an appropriate solution might be to just make decodeMin and decodeMax non-optional with POSITION attributes like we proposed doing with the regular accessor.min and max; it's the only case I can think of where you really need it anyway.

That being said, it probably isn't a bad idea to have a more general way to do matrix math in Cesium. It would open the door down the road to do more complex geometric operations (like finding the circumsphere of a tetrahedron).